canonical-web-and-design/jaas-dashboard
# Rename relations to integrations on ModelDetails

## Done

Rename relations to integrations on ModelDetails

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check integrations appear as expected

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1515

## Screenshots

<img width="816" alt="Screenshot 2020-12-01 at 14 45 04" src="https://user-images.githubusercontent.com/505570/100755188-d2abcf80-33e3-11eb-9f6c-5cd3f6847cd7.png">
